Made servlet and js for file upload

KevinAlamo opened the pull request.

KevinAlamo committed:
Made servlet and js for file upload

conniewu1 reviewed (approved):
Looks good to me! I have the same stickler error that you have at line 3 in your fileuploadform.js file so let me know if you figure out how to fix it!

KevinAlamo committed:
Update fileuploadform.js

Sorry I wasn't able to help in time, I wasn't sure what to do either. I was only able to figure it out because of your question thanks!

KevinAlamo committed:
Update BlobstoreUploadUrlServlet.java

format errors

KevinAlamo committed:
Update fileuploadform.js

KevinAlamo committed:
Update BlobstoreUploadUrlServlet.java

KevinAlamo committed:
Images finally work again! You can now upload your own images onto your user page

KevinAlamo committed:
format changes

KevinAlamo committed:
Merge branch 'master' of https://github.com/KevinAlamo/team-15-codeu into Images_K2

KevinAlamo committed:
Merge branch 'Images_K2' of https://github.com/KevinAlamo/team-15-codeu into Images_K2

KevinAlamo committed:
Update BlobstoreUploadUrlServlet.java

format change

KevinAlamo commented:
Sorry I accidentally added to this pull when I wanted to make a new one, but it doesn't seem to need a review.

KevinAlamo committed:
Update MessageServlet.java

format changes

KevinAlamo committed:
Update MessageServlet.java

more format changes

KevinAlamo committed:
Update MessageServlet.java

KevinAlamo committed:
Update MessageServlet.java

KevinAlamo committed:
Update MessageServlet.java

KevinAlamo committed:
Update MessageServlet.java

KevinAlamo committed:
Update MessageServlet.java

KevinAlamo committed:
Update MessageServlet.java

KevinAlamo committed:
Update MessageServlet.java

KevinAlamo committed:
Update fileuploadform.js
